webpack.common.js

```diff
@@ -108,6 +109,8 @@ module.exports = {
 		new StyleLintPlugin({
 			files: ['**/*.vue'],
 		}),
+		// Make appName & appVersion available as a constant
+		new webpack.DefinePlugin({ appName: JSON.stringify('talk') }),
```

Comment:
where is appVersion ?

also, for the tests you might need to add values in "src/test-setup.js"

Reply:
oops